GAN/VanillaGAN-Torch/dataloader.py

```diff
@@ -0,0 +1,17 @@
+import torchvision.transforms as transforms
+from torch.utils.data import DataLoader
+from torchvision import datasets
+
+def get_loader(batch_size,num_workers):
+
+    batch_size = 100
+    compose = transforms.Compose(
+            [transforms.ToTensor(),
+            transforms.Normalize((.5,), (.5,))
+            ])
+    data = datasets.MNIST(root='./Data', train=True, transform=compose, download=True)
+    data_loader = DataLoader(data, batch_size=batch_size, shuffle=True)
+    #num_batches = len(data_loader)
+
+    return data_loader
+
```

Comment:
remove unnecessary comments and blank lines at the end

Reply:
also use root as parser argument

Reply:
The root meaning the out directory?